src/sync.js

```diff
@@ -0,0 +1,24 @@
+var mubsub = require('mubsub');
+var debug = require('debug')('feathers-sync');
+var mubsub = require('./mubsub');
+var redis = require('./redis');
+
+module.exports = function(adapter, config) {
```

Comment:
Not happy with embedding the arguments. Might be cleaner to just export the un executed adapter and then inject `config` in to the adapter directly when executing with `app.configure(sync({}))`.